documentation/docs/20-core-concepts/10-routing.md

```diff
@@ -12,6 +12,19 @@ At the heart of SvelteKit is a _filesystem-based router_. The routes of your app
 
 Each route directory contains one or more _route files_, which can be identified by their `+` prefix.
 
+Summary of the routing files:
+
+| File name         | Role                                                  | Propagates to sub-dirs | SSR | CSR |
+|-------------------|-------------------------------------------------------|:----------------------:|:---:|:---:|
+| +page.svelte      | HTML page                                             |            ❌           |  ✅  |  ✅  |
+| +error.svelte     | HTML page to render HTTP errors                       |            ✅           |  ✅  |  ✅  |
+| +layout.svelte    | Layout used. Needs at least a <slot /> tag            |            ✅           |  ✅  |  ✅  |
+| +page.js          | Send public data to HTML page                         |            ❌           |  ✅  |  ✅  |
+| +page.server.js   | Send private data to HTML page                        |            ❌           |  ✅  |  ❌  |
+| +server.js        | API endpoint to be called from HTML pages             |            ❌           |  ✅  |  ❌  |
+| +layout.js        | Send public data to HTML page covered by this layout  |            ✅           |  ✅  |  ✅  |
+| +layout.server.js | Send private data to HTML page covered by this layout |            ✅           |  ✅  |  ❌  |
```

Comment:
I haven't formed an opinion about this idea yet, but if we did do it then I think many of the descriptions should be different
```suggestion
|-------------------|-------------------------------------------------------|:----------------------:|:---:|:---:|
| +page.svelte      | Generates HTML page                                   |            ❌           |  ✅  |  ✅  |
| +error.svelte     | Generates HTML page when error encountered             |            ✅           |  ✅  |  ✅  |
| +layout.svelte    | Generates layout HTML                                       |            ✅           |  ✅  |  ✅  |
| +page.js          | Provide data for page via `load`                      |            ❌           |  ✅  |  ✅  |
| +page.server.js   | API endpoint for page. Automatically called             |            ❌           |  ✅  |  ❌  |
| +server.js        | API endpoint to be called manually                    |            ❌           |  ✅  |  ❌  |
| +layout.js        | Provide data for layout via `load`                    |            ✅           |  ✅  |  ✅  |
| +layout.server.js | API endpoint for layout. Automatically called          |            ✅           |  ✅  |  ❌  |
```

Reply:
Not sure about "Generate", but your suggestions are good, I'll make an update and suggest another clearer wording 👍

Reply:
> Also, why do you think it's necessary to specify "Automatically called", while it's already ticked in the SSR column?

By "automatically called", I mean you don't have to write a `load` function and call `fetch` yourself. I don't see what that has to do with SSR